ui/react-components/components/ColorPicker/ColorPicker.module.scss

```diff
@@ -0,0 +1,46 @@
+.colorPickerWrapper {
+  position: relative;
+  display: inline-block;
+  vertical-align: middle;
+  margin: 0;
+  .colorBoxTrigger {
+    span {
+      height: 17px;
+      width: 17px;
+      display: inline-block;
+      float: left;
+      border: 1px solid transparent;
+    }
+    i {
+      float: left;
+      line-height: 19px;
+      padding: 0 2px 0 5px;
+      font-size: 17px;
+      color: #1a1a1a;
+    }
+    margin: 0;
+    overflow: hidden;
+    padding: 3px;
+    border: 1px solid #ddd;
+    border-radius: 3px;
```

Comment:
Instead of adding element styles at the end and child element's styles at the beginning, add elements styles first and child elements styles next. It could be easier to find and traverse the elements styles and then children element styles.

Reply:
Done